evalml/tests/component_tests/test_prophet_regressor.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+from pytest import importorskip
+
+from evalml.model_family import ModelFamily
+from evalml.pipelines.components import ProphetRegressor
+from evalml.problem_types import ProblemTypes
+from evalml.utils.gen_utils import suppress_stdout_stderr
+
+prophet = importorskip('fbprophet', reason='Skipping test because xgboost not installed')
+
+
+def test_model_family():
+    assert ProphetRegressor.model_family == ModelFamily.PROPHET
+
+
+def test_problem_types():
+    assert set(ProphetRegressor.supported_problem_types) == {ProblemTypes.TIME_SERIES_REGRESSION}
+
+
+def test_fit_predict_ts(ts_data):
+    X, y = ts_data
+
+    def build_prophet_df(X, y=None):
```

Comment:
Agree with Freddy, could be nice to make the `build_prophet_df` method a util method so you can use in both implmentation and tests